synth.py

```diff
@@ -40,7 +40,7 @@
 templates = common_templates.node_library(
   source_location='build/src'
 )
-s.copy(templates)
+s.copy(templates, exclude='.jsdoc.js')
```

Comment:
whoops this should be:

```
excludes=['.jsdoc.js`]
```

I'll make an effort to remove the necessity of this step early this week.

Reply:
My bad, thanks @bcoe! Sending a fix now.